.cirrus.yml

```diff
@@ -0,0 +1,65 @@
+aws_credentials: ENCRYPTED[340e4a4f5dbf62ed509c2c0e638b45f7ca7e9ca2d141377ec59ef2cd032863e65f4d2542b8a93007799def51663ae411]
+
+env:
+  MAVEN_CONFIG: ""
+  MAVEN_OPTS: -Xmx512M -XX:+ExitOnOutOfMemoryError
+  MAVEN_SKIP_CHECKS_AND_DOCS: -Dair.check.skip-all=true -Dmaven.javadoc.skip=true
+  MAVEN_FAST_INSTALL: -DskipTests $MAVEN_SKIP_CHECKS_AND_DOCS -B -q -T 1C
+
+webui_checks_task:
+  ec2_instance:
+    image: ami-08d489468314a58df
+    type: t2.micro
+    region: us-east-1
+  container:
+    image: node:latest
+  check_script: presto-main/bin/check_webui.sh
+
+maven_checks_task:
+  ec2_instance:
+    image: ami-08d489468314a58df
+    type: m5.xlarge
+    region: us-east-1
+  container:
+    image: maven:3-jdk-8
```

Comment:
You probably want to remove `container` from both of the tasks. This is the reason why `aws_credentials` got erased. Will try to make parser a bit smarter and throw on such cases when the instance is overridden on the same yaml level.